example_tb/core/Makefile

```diff
@@ -252,6 +252,34 @@ interrupt-vsim-run-gui: vsim-all interrupt/interrupt.hex
 interrupt-vsim-run-gui: ALL_VSIM_FLAGS += "+firmware=interrupt/interrupt.hex"
 interrupt-vsim-run-gui: vsim-run-gui
 
+# Compile and run HW-loops test
```

Comment:
Can this be moved to core-v-verif right away? We are trying to keep this Makefile as minimal as possible (ideally only 'hello world')